.ci-scripts/test-bootstrap.sh

```diff
@@ -3,6 +3,11 @@
 ./ponyup-init.sh --repository=nightlies
 
 export PATH=$HOME/.local/share/ponyup/bin:$PATH
+
+if [ -n "${PLATFORM}" ]; then
+  ponyup default "${PLATFORM}"
+fi
```

Comment:
@Theodus so I think this is the most reasonable path forward. If PLATFORM env is defined, we run default for it. otherwise, we don't.

Reply:
Seems good to me